test/parallel/test-module-readonly.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+if (process.platform == 'win32') {
```

Comment:
Since this is currently a Windows only test, you can do:

```
if (!common.isWindows) {
  common.skip('test only runs on Windows');
}
```

This will skip the test on other platforms and exit the process.